DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/Index.cshtml

```diff
@@ -10,6 +10,10 @@
   ViewData["Application"] = "Tracking System";
   ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
   ViewData["HeaderPathName"] = "Tracking System";
+  var summaryText = string.IsNullOrEmpty(Model.ActivityFilterModel.CustomisationName) ?
+    Model.ActivityFilterModel.CourseCategoryName : Model.ActivityFilterModel.CustomisationName;
+  var summaryType = string.IsNullOrEmpty(Model.ActivityFilterModel.CustomisationName) ?
+    "courses" : "course";

```

Comment:
This should be based not on filter, but on admin course category. We are displaying similar text on the Centre Dashboard (see image below).

If the user has only a certain course category permission, they should never see a course category filter on the card and only be able to filter based on a course. If the user can view "All" categories (categoryId = 0), they can filter based on category OR based on a course. 

![image](https://user-images.githubusercontent.com/40016796/131661601-25c3fdd4-2bec-4d2e-93f7-63780e55c6ca.png)